docs/pages/admin-guides/management/guides/migrating-iam-ic-from-okta-to-teleport.mdx

```diff
@@ -0,0 +1,334 @@
+
+![Migration Starting Point](../../../../img/identity-center/ic-migrate-start.png)
+
+#### Midway
+
+This is the point where you can start migrating control of users and groups from
+Okta to Teleport. At this stage:
+
+ - Okta provides SSO login for Identity Center
+ - Okta manages a subset of the Identity Center group membership (selected via Push Groups)
+ - Okta and Teleport both control user provisioning via SCIM
+ - Teleport manages a second subset of Identity Center groups (selected via group filters during plugin installation)
+ - Teleport controls IC Group Account Assignments for the Identity Center groups under its control
+ - Teleport controls direct IC User Account Assignments for the Identity Center users under its control
+
+![Migration Mid Point](../../../../img/identity-center/ic-migrate-mid.png)
+
+#### Ending point
+
+Once Teleport is in control of all the users and groups you want to provision
+into Identity Center, the Okta Provisioning and Push Groups functions can be
+retired, leaving Okta only providing SSO login.
+
+![Migration End Point](../../../../img/identity-center/ic-migrate-end.png)
+
+## Prerequisites
+
+ - A working Teleport cluster
+ - An AWS role configured as per the [Identity Center guide](./aws-iam-identity-center.mdx)
+   for the integration to use
+ - AWS credentials configured for the Teleport auth process to pick up and use 
+   (e.g. as environment vars, system profiles, etc)
+ - An Okta API token, with the following privileges:
+   - View users and their details
+   - View groups and their details
+   - View applications and their details
+ - The AWS Identity Center ARN, AWS region, SCIM base address and SCIM bearer token
+
+## Step 1/7. Install Okta SAML connector
+
+Install Okta SAML connector into Teleport as per the Teleport 
+[Okta as an SSO provider](../../access-controls/sso/okta.mdx) guide.
+
+<Admonition type="note">
+ For the integration to function properly, both AWS Identity Center and Teleport must view the
+ same user set which can be achieved by using the same Okta SAML application for both Identity
+ Center and Teleport SAML connector.
+
+ If this is not possible, the most flexible approach we've found is having an
+ an Okta Group for those users that should have IC access, and then assigning
+ both the Okta Identity Center App and the Okta SAML App used for Teleport SAML
+ connector to that group. This ensures that the same user set is visible across
+ both applications.
+</Admonition>
+
+You will need the Teleport SAML Connector Name and Okta SAML App ID in the next
+step.
+
+## Step 2/7. Install the Teleport Okta integration
+
+We will be using a very limited subset of the Teleport Okta integration in this 
+deployment, disabling all features except periodic user synchronization. This
+configuration is not currently supported by the normal installation UI, so we
+will have to use `tctl` to install it:
+
+```console
+$ tctl plugins install okta \
```

Comment:
Is there a command reference for this command and "install awsic" used below? I think we should have it to explain all the flags: https://goteleport.com/docs/reference/cli/tctl/.

Here (and below where "install awsic" is used), it might be helpful to briefly explain what effect these flags have as well.

Reply:
Added in #52878